tests/group_functions/group_broadcast.h

```diff
@@ -164,13 +164,12 @@ void broadcast_sub_group(sycl::queue& queue) {
 
         // To simplify the test, we are only checking the first sub-group
         if (sub_group.get_group_id()[0] == 0) {
-          // Find local id of first, last and some third sub-group item in
-          // between. Will be used to check different combinations of
+          // Find local id of a leader, last and some other work-item in the
```

Comment:
```suggestion
          // Find local id of the leader, last and some other work-item in the
```
"the leader" in the sense of the `.leader()`?